index.html

```diff
@@ -47,132 +45,38 @@ <h1>Johnson Macharia</h1>
-            Project name goes here.
-          </p>
-          <ul class="tech-list">
-            <li>html/css</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
-
-      <div class="project-card project-card-2">
-        <img class="preview" src="icons/Rectangle 21 (1).png">
-        <div class="project-infor">
-          <p class="project-name">
-            Project name goes here.
-          </p>
-          <ul class="tech-list">
-            <li>html/css</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
 
-      <div class="project-card project-card-3">
-        <img class="preview" src="icons/Rectangle 21 (2).png">
-        <div class="project-infor">
-          <p class="project-name">
-            Project name goes here.
-          </p>
-          <ul class="tech-list">
-            <li>html/css</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
-
-      <div class="project-card project-card-4">
-        <img class="preview" src="icons/Rectangle 21 (3).png">
-        <div class="project-infor">
-          <p class="project-name">
-            Project name goes here.
-          </p>
-          <ul class="tech-list">
-            <li>html/css</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
-
-      <div class="project-card project-card-5">
-        <img class="preview" src="icons/Rectangle 21 (4).png">
-        <div class="project-infor">
-          <p class="project-name">
-            Project name goes here.
-          </p>
-          <ul class="tech-list">
-            <li>html/css</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
-
-      <div class="project-card project-card-6">
-        <img class="preview" src="icons/Rectangle 21 (5).png">
-        <div class="project-infor">
-          <p class="project-name">
-            Project name goes here.
-          </p>
-          <ul class="tech-list">
-            <li>html/css</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
+    <section id="project-container" class="">
+      <h2>Projects</h2>
+      <div class="dec-2"><img class="decoration" src="icons/image-geometry_2 1.png"></div>
+      <div class="dec-3"><img class="decoration" src="icons/image-geometry_3.png"></div>
```

Comment:
- [ ] Kindly add icons missing in your buttons 

![image](https://github.com/JohnteFresh/Portfolio-Development/assets/93630700/ff152eb2-c880-41ba-8640-940aac37e94e)

- [ ] Kinldy add the different hover effects to the buttons as it's on the Figma design

![image](https://github.com/JohnteFresh/Portfolio-Development/assets/93630700/fd448748-f08b-42bc-bff6-b09bcda89a38)